ue4docker/dockerfiles/diagnostics/20gig/windows/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# escape=`
+ARG BASETAG
+FROM mcr.microsoft.com/windows/servercore:${BASETAG} as intermediate
+
+# Add a sentinel label so we can easily identify intermediate images
+LABEL com.adamrehn.ue4-docker.sentinel="1"
+
+# Create a file slightly over 20GiB
+RUN fsutil file createnew C:\bigfile.txt 22548578304
+
+FROM mcr.microsoft.com/windows/servercore:${BASETAG}
+
+# Add a sentinel label so we can easily identify intermediate images
+LABEL com.adamrehn.ue4-docker.sentinel="1"
+
+# Copy more than 20GiB between docker layers
+COPY --from=intermediate C:\bigfile.txt C:\\bigfile.txt
```

Comment:
Different backslashes on the two filenames. Is that meaningful?

Reply:
> Is that meaningful?

Nope, fixed